server/database/middleware/index.js

```diff
@@ -2,10 +2,16 @@ const express = require('express');
 const morgan = require('morgan');
 const helmet = require('helmet');
 const cors = require('cors');
+const URL = 'http://localhost:777'
```

Comment:
did you configure your front end to run on 777? Otherwise you can't access it from port 3000 by default on react...Looking at your React Package.JSON it seems that your front end runs on 778 instead..